src/communication_mode.c

```diff
@@ -0,0 +1,29 @@
+// Copyright 2025 Shift Cryptosecurity AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "communication_mode.h"
+
+#include "memory/memory_shared.h"
+
+static bool _usb_hww_request_seen = false;
+
+void communication_mode_enable_usb(void)
```

Comment:
I would prefer if the two functions match. So more something like

`disable_ble` / `ble_enabled` or `enable_usb` / `usb_enabled`.

I think I also prefer to suffix the verb, so `usb_disable` / `usb_enabled`

With the current naming you have to implicitly understand that enabling usb disables ble.

Reply:
Renamed :+1: